tests/variable_test.py

```diff
@@ -102,6 +102,14 @@ def test_0D_scalar_access():
     assert var.values == 1.2
 
 
+@pytest.mark.parametrize('dtypes', (('int32', np.int32), ('int64', np.int64),
+                                    ('float32', np.float32), ('float64', np.float64),
+                                    ('bool', np.bool_), ('datetime64', np.datetime64)))
```

Comment:
I remember that using `np.bool_` was considered bad and problematic. Shouldn't we use Python's bool?

Reply:
Do you know why? A quick search only told me that `np.bool_(True) is True` gives `False` which can be surprising. But it also told me to never use `is True` or `is False` anyway.

I am fine with switching back to builtin `bool` because we don't need to distinguish sizes or anything like that with `bool`. Just wondering if you know more about it?

Reply:
I forgot unfortunately.